src/scripts/make-webpack-config.ts

```diff
@@ -108,8 +108,14 @@ export default function(
 			},
 		});
 		if (config.assetsDir && webpackConfig.plugins) {
+			const copyPatterns = {
+				patterns: castArray(config.assetsDir).map(dir => ({ from: dir })),
+			};
 			webpackConfig.plugins.push(
-				new CopyWebpackPlugin(castArray(config.assetsDir).map(dir => ({ from: dir })))
+				// since we don't have the type of copy-webpack-plugin@6.0
```

Comment:
To make it more visible:

```suggestion
				// FIXME: Since we don't have the type of copy-webpack-plugin@6.0
```